images/macos/tests/Ndk.Tests.ps1

```diff
@@ -0,0 +1,32 @@
+
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
+Import-Module "$PSScriptRoot/../helpers/Tests.Helpers.psm1"
+Import-Module "$PSScriptRoot/../software-report/SoftwareReport.Android.psm1"
+
+$os = Get-OSVersion
```

Comment:
I suggest to rename both `Describe "NDK"` and this file to be more specific.

`Ndk` is not clear enough, it just means Native Development Kit and could be related to any tool. Not only Xamarin.
Let's name both as `Xamarin NDK`